### PR TITLE
Memory limit for CLI

### PR DESCRIPTION
**Pull request by AronNovak**
#383

### CODE REVIEW

**Comment by AronNovak** [comment]
image

File: .ddev/config.local.yaml.example [hunk 1]
```diff
@@ -42,3 +42,10 @@ hooks:
 
     # Login as admin.
     - exec-host: ddev login
+
+    # Make sure PHP CLI has no memory limit, this is
+    # especially for unit testing.
+    # Web requests should be still limited, as the
+    # hosting environment won't provide unlimited
+    # memory either.
+    - exec: perl -pi -e 's/memory_limit.*$/memory_limit = -1/' /etc/php/${DDEV_PHP_VERSION}/cli/conf.d/*.ini
```
Comment (amitaibu) [inline .ddev/config.local.yaml.example:51]: Can we do something similar to this - https://stackoverflow.com/a/72787008/750039 ?

Comment (AronNovak) [reply .ddev/config.local.yaml.example:51]: I'd not do that. It affects everything, web requests and CLI too. For generating web pages, we never want to allow that amount of memory to be consumed and I think it's better to enforce it early and not only on Pantheon.

See this:  https://github.com/drud/ddev/issues/1825#issuecomment-529964728 - it's @rfay approved solution :)

Comment (amitaibu) [reply .ddev/config.local.yaml.example:51]: Got it. Let's add the link on top of the code then

**Review by amitaibu — APPROVED** [review]

**Pull request merged**